### PR TITLE
Make release announce

### PR DESCRIPTION
**Pull request by abadger**
##### SUMMARY
Script to generate release announcements

##### ISSUE TYPE
- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request

##### ADDITIONAL INFORMATION
Here's a sample run from today.  The first, small blob of text is for publishing to twitter.  The rest is what would get sent to the mailing list.

```
$ ./release-announcement.py --name 'Toshio Kuratomi' --version 2.7.9 --version 2.6.15 --version 2.5.15


@ansible 2.7.9, 2.6.15, and 2.5.15 have been released! Get them on PyPI: pip install ansible==2.7.9, https://releases.ansible.com/ansible/, the Ansible PPA on Launchpad, or GitHub. Happy automating!




To: ansible-devel@googlegroups.com, ansible-project@googlegroups.com, ansible-announce@googlegroups.com
Subject: New Ansible release 2.7.9, 2.6.15, and 2.5.15

Hi all- we're happy to announce that the general release of Ansible 2.7.9,
2.6.15, and 2.5.15 is now available!


How do you get it?
------------------

$ pip install ansible==2.7.9 --user
or
$ pip install ansible==2.6.15 --user
or
$ pip install ansible==2.5.15 --user

The tar.gz of the release can be found here:

* 2.7.9
  https://releases.ansible.com/ansible/ansible-2.7.9.tar.gz
  SHA256: 6ee59d561c206f82e842f3aae4b56fc9a8b83ffecba4cab33bdb56f781717ea7
* 2.6.15
  https://releases.ansible.com/ansible/ansible-2.6.15.tar.gz
  SHA256: 0d7a6ef42d1f464daafcf8305352e4d7218c7f71cc78188aa1c03a9f758a92d0
* 2.5.15
  https://releases.ansible.com/ansible/ansible-2.5.15.tar.gz
  SHA256: ffbc2b093290c72a0a79be457ba47c57ecce7f32d65418fadc064ba1c3763cf1


What's new in 2.7.9, 2.6.15, and 2.5.15
---------------------------------------

These releases are maintenance release containing numerous bugfixes. The full
changelogs are at:

* 2.7.9
  https://github.com/ansible/ansible/blob/stable-2.7/changelogs/CHANGELOG-v2.7.rst
* 2.6.15
  https://github.com/ansible/ansible/blob/stable-2.6/changelogs/CHANGELOG-v2.6.rst
* 2.5.15
  https://github.com/ansible/ansible/blob/stable-2.5/changelogs/CHANGELOG-v2.5.rst


What's the schedule for future maintenance releases?
----------------------------------------------------

Future maintenance releases will occur approximately every 3 weeks.  So expect
the next one around 2019-04-11.


Porting Help
------------

We've published a porting guide at
https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_2.7.html to
help migrate your content to 2.7.


If you discover any errors or if any of your working playbooks break when you
upgrade to 2.7.9, please report the regression via
https://github.com/ansible/ansible/issues/new/choose  In your issue, be sure to
mention the Ansible version that works and the one that doesn't.

Thanks!

-Toshio Kuratomi

```

### CODE REVIEW

**Review by gundalow — COMMENTED** [review]
Yay for automation.
For the email should we generate the email subject line as well?

**Comment by acozine** [comment]
Automating the typos away is a great idea!

The phrase `if you see any regressions from playbooks which work on previous versions` is a bit geek-speak-ish. If we're crafting this message for a general audience, including new users and non-programmers, I'd change that. Something like `or if any of your working playbooks break when you upgrade to 2.x.x,` is simpler - then we can use the more technical term as we continue, `please report the regression in an issue on GitHub. In your issue, be sure to mention the Ansible version that works and the one that doesn't.`

**Comment by gundalow** [comment]
Report issues via https://github.com/ansible/ansible/issues/new/choose

**Comment by abadger** [comment]
Thanks for the review.  I think I've incorporated all the feedback.  I'll update the original comment with the new text.

File: test/sanity/compile/python2.6-skip.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Only run by release engineers who can be asked to have newer Python3 on their systems"
```
Comment (mattclay) [inline test/sanity/compile/python2.6-skip.txt]:
```suggestion
# Only run by release engineers who can be asked to have newer Python3 on their systems
```

File: test/sanity/compile/python2.7-skip.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Only run by release engineers who can be asked to have newer Python3 on their systems"
```
Comment (mattclay) [inline test/sanity/compile/python2.7-skip.txt]:
```suggestion
# Only run by release engineers who can be asked to have newer Python3 on their systems
```

File: test/sanity/compile/python3.5-skip.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Only run by release engineers who can be asked to have newer Python3 on their systems"
```
Comment (mattclay) [inline test/sanity/compile/python3.5-skip.txt]:
```suggestion
# Only run by release engineers who can be asked to have newer Python3 on their systems
```

**Comment by abadger** [comment]
Thanks all for reviewing :-)

**Pull request merged**